### PR TITLE
Qr scanner

### PR DESCRIPTION
**Pull request by dmroberts302**
I Added QR code scanner functionality in route /checkInOut. Is it fine the QR code scanner does not load immediately, it takes like 1s to render?

### CODE REVIEW

**Comment by JackTNorris** [comment]
Taking a second to render is perfectly fine

File: src/screens/CheckInOutScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+    display: block;
+    margin: 0px 2px;
+    cursor: pointer;
+    border-radius: 40px;
+    font-family: monospace;
+    font-weight: 700;
+    margin: auto;
+`;
+
+const camStyle = {
```
Comment (JackTNorris) [inline src/screens/CheckInOutScreen.tsx:23]: Can we move this to a CSS file?

File: src/screens/CheckInOutScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
+import { FunctionComponent } from 'react';
+import { useState, useRef } from 'react';
+import { QrReader } from 'react-qr-reader';
+import styled from 'styled-components';
+import React from 'react';
+
+const Custom = styled.button`
```
Comment (JackTNorris) [inline src/screens/CheckInOutScreen.tsx:7]: Go with an approach using the MUI buttons like done [here](https://github.com/uark-acm/bho-frontend/blob/a0e43077d1c46360685a1a9e8ef27abdb82c972f/src/components/ViewItemModal.tsx#L79-L87):

**Review by JackTNorris — CHANGES_REQUESTED** [review]

File: src/screens/CheckInOutScreen.tsx (hunk 1 above)
Comment (dmroberts302) [reply src/screens/CheckInOutScreen.tsx]: When looking back at the codeI noticed cam style had some redundant css which i removed.

File: src/screens/CheckInOutScreen.tsx (hunk 2 above)
Comment (dmroberts302) [reply src/screens/CheckInOutScreen.tsx]: I have added the MUI Button to my code and it works perfectly fine!

File: src/screens/CheckInOutScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
...
+                    width: '45%',
+                    margin: 'auto',
+                    marginTop: '-20px',
+                    marginBottom: '-25px',
+                }}
+            />
+            <Typography align="center">
+                <Button
+                    variant="contained"
+                    style={{
```
Comment (JackTNorris) [inline src/screens/CheckInOutScreen.tsx:43]: _nit_: maybe move this to a css file if possible. Atm, I'll approve, but see if you can move it to a CSS file before merging

**Review by JackTNorris — APPROVED** [review]

File: src/screens/CheckInOutScreen.tsx (hunk 3 above)
Comment (dmroberts302) [reply src/screens/CheckInOutScreen.tsx:43]: Hey Jack for some reason its not letting me add external css to a Material UI button.

**Pull request merged**